app/components/NoSuggestionsPanel/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Icon from '../Icon'
+import { IntlMixin } from 'react-intl'
+
+/**
+ * Generic panel showing an icon and message, to
+ * use when there are no suggestions to display.
+ */
+let NoSuggestionsPanel = React.createClass({
```

Comment:
I agree, there does not seem to be anything about the code that would make it only suitable for the suggestions panel. Maybe something like `CenteredMessagePanel`